src/Worker.Extensions.DurableTask/TaskEntityDispatcher.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask;
```

Comment:
Any thoughts on namespace here? Should this be `Microsoft.DurableTask.Entities` to go with all the other entity types?

Reply:
In the user code this class always appear together with `EntityTriggerAttribute` so maybe `Microsoft.Azure.Functions.Worker` makes sense so they don't need an extra using statement.

Reply:
I see most extensions do put their attributes in that namespace. Not a fan as it bloats the namespace, but yeah we can follow that example.